docs/generate.rst

```diff
@@ -121,6 +121,14 @@ HPCs. To change the run time per benchmark, simply use the ``--time`` option::
 
 This would run all benchmarks for a total of five minutes.
 
+Changing the job name
+---------------------
+
+If you want your benchmark jobs to have specific names, use the ``--job-name`` option::
+
+  mdbenchmark generate --job-name my_benchmark
+
 .. _modules: https://linux.die.net/man/1/module
 .. _draco: https://www.mpcdf.mpg.de/services/computing/draco
 .. _hydra: https://www.mpcdf.mpg.de/services/computing/hydra
+.. _datreant: https://www.datreant.org/
```

Comment:
I added a short section on datreant which seems to be gone now. Should we leave this in as reference even though it is not mentioned in the text?

Reply:
I don't think it is necessary to say anything about `datreant` at the current point.